### PR TITLE
enable to scroll lcb-tabs when watch many tabs

### PR DESCRIPTION
**Pull request by muddydixon**
## Abstract
In this p-r, enable to scroll lcb-tabs when watching many rooms.

### CODE REVIEW

**Comment by tily** [comment]
+1

File: templates/chat.html [hunk 1]
```diff
@@ -76,13 +76,13 @@
...
                 </span>
             </div>
-            <ul class="lcb-tabs">
-                <li data-id="list" class="lcb-tab lcb-tab-home">
-                    <a href="#!/">
-                        <i class="fa fa-th"></i>
-                    </a>
-                </li>
-            </ul>
+            <div data-id="list" class="lcb-tab lcb-tab-home">
```
Comment (hhaidar) [inline templates/chat.html]: Why are these divs? Also lcb-tab-home class should be on the first item?

File: templates/chat.html [hunk 2]
```diff
@@ -76,13 +76,13 @@
...
+              <a href="#!/">
```
Comment (hhaidar) [inline templates/chat.html]: Whitespace for html should be 4 spaces

File: media/js/views/panes.js [hunk 3]
```diff
@@ -49,6 +51,12 @@
         update: function(room) {
             this.$el.find('.lcb-tab[data-id=' + room.id + '] .lcb-tab-title').text(room.get('name'));
         },
+        updateHeight: function(){
+          var padding = $('.lcb-account-button').height() +
```
Comment (hhaidar) [inline media/js/views/panes.js]: We can use CSS to handle layout. We have the power :D

Comment (muddydixon) [inline media/js/views/panes.js]: If you know how to adjust height of lcb-tabs by CSS, please teach me.

**Comment by muddydixon** [comment]
I fixed some points.

**Comment by hhaidar** [comment]
@muddydixon do you mind checking #445? That should fix the issue with just CSS.

**Comment by muddydixon** [comment]
no problem. #445 seems better.

**Pull request closed without merging**